integration/rstudio-connect.gcfg

```diff
@@ -6,7 +6,8 @@
 ; email. The system will not be able to send administrative email until this
 ; setting is configured.
 ;
-; SenderEmail = account@company.com
+SenderEmail = no-reply@example.com
+EmailProvider = none
```

Comment:
> When none is used, if legacy email settings were previously configured in the Posit Connect administrator interface, they will be used instead. Otherwise, all the email features will be disabled.

Very curious to see if this will work. Down to give a shot though!

Reference: https://docs.posit.co/connect/admin/appendix/configuration/#Server.EmailProvider

Reply:
The text "legacy email settings" references really old DB-backed email settings. They never apply anymore. I'll make sure that the `Server.EmailProvider` settings text is updated to reflect this.